<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>M328Dialog</class>
 <widget class="QDialog" name="M328Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>461</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>Lock and Fuse Bit Edit m328</string>
   </property>
  </widget>
  <widget class="QFrame" name="extFuseFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QLabel" name="extFuseLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Extended Fuse Bits</string>
    </property>
   </widget>
   <widget class="QComboBox" name="brownoutBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>38</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Disabled</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.8V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.7V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4.3V</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="brownoutLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Brownout Level</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="highFuseFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>351</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QLabel" name="highFuseLabel_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>High Fuse Bits</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="resetDisableBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>37</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Disables the external reset pin for use as a general port</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>External Reset Disable  </string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="debugWIREBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>57</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enables the debugWire capability</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>debugWIRE Enable</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="enableSerialBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>77</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enables the use of serial (SPI) programming of the target</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Enable Serial Programming</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="watchdogOnBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>97</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Prevents the watchdog timer from being turned off</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Watchdog Timer always on</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="preserveEEPROMBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>117</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Keeps the EEPROM contents following a chip erase action</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Preserve EEPROM on reset</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="bootSizeBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>143</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string> 256 Words</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> 512 Words</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1024 Words</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2048 Words</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="bootResetCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>When checked it will start in the bootloader section on reset, otherwise it starts at location 0000</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Bootloader Reset Vector</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bootSizeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>147</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Bootloader Size</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="fuseFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>351</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QCheckBox" name="clockOutBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Output the clock signal on pin PB0</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Clock Output</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="clockDivide8Box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Divide down the clock frequency by 8</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Divide Clock by 8</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="fuseLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Fuse Bits</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="startupTimeBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="startupTimeBox_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="clockSourceBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="clockSourceBox_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="clockSourceBox_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="clockSourceBox_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="startupTimeLabel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>This depends on the choice of clock source. Refer to the datasheets. The settings are not inverted and match those of the datasheet.</string>
    </property>
    <property name="text">
     <string>Startup Time Selection</string>
    </property>
   </widget>
   <widget class="QLabel" name="clockSourceLabel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Clock Source Selection.. Refer to the datasheets. The settings are not inverted and match those of the datasheet.</string>
    </property>
    <property name="text">
     <string>Clock Source Selection</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="warningLabel_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Labelled checkbox settings are the inverse of those given in the datasheet
except for the Startup Time and Clock Source check bits.</string>
   </property>
  </widget>
  <widget class="QFrame" name="lockFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>351</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QLabel" name="memoryLockLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Memory Lock</string>
    </property>
   </widget>
   <widget class="QComboBox" name="appLockBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>66</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Selects the type of memory locking in the application memory space.</string>
    </property>
    <item>
     <property name="text">
      <string>Full SPM and LPM Access</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>No SPM Write Access</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>No LPM Read Access</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>No SPM Write/LPM Read Access</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="memoryLockBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>36</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Selects the type of locking of memory, either none, read only or read/write. Fuse bits are locked in the latter two modes.</string>
    </property>
    <item>
     <property name="text">
      <string>No Memory Lock</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Memory/Fuse Write Lock</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Memory/Fuse/BootLock Read and Write Lock</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="appLockLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Application Lock</string>
    </property>
   </widget>
   <widget class="QLabel" name="bootLockLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Bootloader Lock</string>
    </property>
   </widget>
   <widget class="QLabel" name="lockLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Lock Bits</string>
    </property>
   </widget>
   <widget class="QComboBox" name="bootLockBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>96</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Selects the type of memory locking in the bootloader memory space.</string>
    </property>
    <item>
     <property name="text">
      <string>Full SPM and LPM Access</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>No SPM Write Access</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>No LPM Read Access</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>No SPM Write/LPM Read Access</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="lockWriteButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Write Locks</string>
   </property>
  </widget>
  <widget class="QPushButton" name="extFuseWriteButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Write EFBs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="highFuseWriteButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Write HFBs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fuseWriteButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>500</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Write FBs</string>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>550</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Startup time and
clock source bits
are as shown in
the datasheet</string>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lock bits are
cleared by a Chip
Erase (but not in
a bootloader).</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>610</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
